index.bs

```diff
@@ -44,6 +44,105 @@ url: https://dom.spec.whatwg.org/#ref-for-concept-getelementsbytagname; type: in
     }
 </style>
 
+<h2 id="api-principles">Principles behind design of Web APIs</h2>
+
+<h3 id="safe-to-browse">It should be safe to visit a web page</h3>
+
+Hyperlinks, links from one page to another, are one of the foundations of the Web.
+
+Following a link, or visiting a web page, should be safe.
+Users who are following a link to a web page they haven't seen before
+should not have to fear for the security of their computer
+or for essential aspects of their privacy.
+(One caveat here is that users of the Web
```

Comment:
I'm not sure I understand this caveat. I agree with the general principle, of course, but I'm not sure what it adds here - to me it reads as though there can/should be escape hatches from users' privacy protections in cases where the user is harming others, which I'm sure is not the intended reading.

Reply:
So the caveat is really to the sentence before the quote -- the idea that following a link should always be safe.  Taking that position absolutely would imply that we're taking the position that governments shouldn't punish people for, say, viewing child pornography.  And I don't think that's a position that I'd like to take.  So I want some way to say that we want it to be safe from a technical perspective but not necessarily from a legal one.

(At the same time, there's an entirely separate discussion about mechanisms that governments want to use to enforce some legal rules, and what the downsides of those mechanisms are, and while some of that discussion is relevant to the TAG, I'd like to avoid getting into it in this section.)

I should think more about a better way to word this.